src/foam/util/Emails/EmailsUtility.java

```diff
@@ -57,6 +59,19 @@ public static void sendEmailFromTemplate(X x, User user, EmailMessage emailMessa
       }
     }
 
+    SupportConfig supportConfig = theme.getSupportConfig();
+
+    // Set ReplyTo, From, DisplayName in emailMessage 
+    EmailConfig emailConfig = supportConfig.getEmailConfig();
+    // REPLY TO:
+    emailMessage.setReplyTo(emailConfig.getReplyTo());
+
+    // DISPLAY NAME:
+    emailMessage.setDisplayName(emailConfig.getDisplayName());
+
+    // FROM:
+    emailMessage.setFrom(emailConfig.getFrom());
```

Comment:
Please check for not empty before set. eg,

    if ( ! SafetyUtil.isEmpty(emailConfig.getReplyTo()) ) {
        emailMessage.setReplyTo(emailConfig.getReplyTo());
    }